foreshadow/concrete/internals/intents/__init__.py

```diff
@@ -0,0 +1,9 @@
+"""Intents."""
+
+from foreshadow.concrete.internals.intents.base import BaseIntent
```

Comment:
Not sure if this is already somewhere else in this CR but why are intents in concrete

Reply:
yeappers moving.